package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mobile-device",
   "widgetName": "MobileDevice",
-  "version": "1.0.1",
+  "version": "1.2.0",
```

Comment:
version should only going up on point, mendix appstore does not allow gaps.
should be 1.1.0

Reply:
Done